ccnmtldjango/template/+package+/main/tests/test_views.py

```diff
@@ -33,7 +33,7 @@ def setUp(self):
 
     def test_page(self):
         r = self.c.get("/pages/section-1/")
-        self.assertEqual(r.status_code, 200)
+        self.assertEqual(r.status_code, 403)
```

Comment:
Ah, okay. I think the answer is yes - the gate check should come first. Otherwise, the code precludes a public/anonymous pagetree site a la Pedialabs Overview section. (http://pedialabs.ccnmtl.columbia.edu/pages/public/overview). Does that seem right?